aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/TransferOperations.kt

```diff
@@ -119,9 +118,8 @@ internal object TransferOperations {
             if (TransferState.isPaused(transferRecord.state)) {
                 if (transferRecord.isMultipart == 1) {
                     abortMultipartUploadRequest(transferRecord, pluginKey, workManager)
-                } else {
-                    nextState = TransferState.CANCELED
                 }
+                nextState = TransferState.CANCELED
```

Comment:
Is this change right being moved out of the else block?

Reply:
thats correct, because paused multipart transfer will be directly cancelled and abort request will be queued.